account_statement_base/views/account_bank_statement.xml

```diff
@@ -0,0 +1,117 @@
+                        <button
+                            name="open_entries"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-bars"
+                        >
+                            <div class="o_stat_info">
+                                <span class="o_stat_text">Journal Items</span>
+                            </div>
+                        </button>
+                    </div>
+                    <div class="oe_title oe_inline">
+                        <label for="name" />
+                        <h1>
+                            <field name="name" placeholder="e.g. BNK/2021/0001" />
+                        </h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="date" />
+                            <field
+                                name='company_id'
+                                options="{'no_create': True}"
+                                groups="base.group_multi_company"
+                            />
+                            <field name="currency_id" invisible="1" />
+                            <field name="journal_id" invisible="1" />
+                        </group>
+                        <group>
+                            <label for="balance_start" />
+                            <div>
+                                <field name="balance_start" class="oe_inline" />
+                            </div>
+                            <label for="balance_end_real" />
+                            <div>
+                                <field name="balance_end_real" class="oe_inline" />
+                            </div>
+                        </group>
+                    </group>
+                    <field
+                        name="line_ids"
+                        context="{'default_journal_id': journal_id}"
+                    />
+                    <group
+                        class="oe_subtotal_footer oe_right"
+                        name="sale_total"
+                        col="1"
+                    >
+                        <field
+                            name="balance_end"
+                            class="oe_subtotal_footer_separator"
+                        />
+                    </group>
+                    <field name="attachment_ids" widget="many2many_binary" />
+                    <div class="oe_clear" />
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <!-- Add form to view mode. -->
+    <record id="account.action_bank_statement_tree" model="ir.actions.act_window">
+        <field name="res_model">account.bank.statement</field>
+        <field name="view_mode">list,form,pivot,graph</field>
+    </record>
+    <record id="account.action_view_bank_statement_tree" model="ir.actions.act_window">
+        <field name="view_mode">tree,form,pivot,graph</field>
```

Comment:
```suggestion
        <field name="view_mode">list,form,pivot,graph</field>
```

Reply:
have conditions to see this action in [here](https://github.com/odoo/odoo/blob/33e32433528025625efa2b87839b1f6fd3bb1f9b/addons/account/views/account_journal_dashboard_view.xml#L70) 
![image](https://github.com/user-attachments/assets/2c7a0c3a-3bcd-43a4-ae67-7d432f2a83d9)